### ISSUE TITLE
pfnServerCmd argument string length limited to 255 characters

### ISSUE THREAD

**Issue by SNMetamorph** [FWGS/xash3d-fwgs#1301]
Argument string length is limited, but we don't have to limit it.
https://github.com/FWGS/xash3d-fwgs/blob/ea2a8b6785247ef6eb39a6e2241568038514292a/engine/client/cl_game.c#L1728-L1740

We can do something like this
```c
static int GAME_EXPORT pfnServerCmd( const char *szCmdString )
{
	if( !COM_CheckString( szCmdString ))
		return 0;

	// just like the client typed "cmd xxxxx" at the console
	Cbuf_AddText( "cmd " );
	Cbuf_AddText( szCmdString );
	Cbuf_AddText( "\n");

	return 1;
}
```

**Comment by a1batross** [comment]
GoldSrc actually has 2048 bytes limit, which is kinda a lot, and we can just ignore the limit and pass it directly to Cbuf_AddText as you suggested.

**Comment by a1batross** [comment]
The funny thing is, we don't have that limitation in ServerCmdUnreliable.

Actually, we probably can just write clc_stringcmd directly to netchan instead. =/

**Comment by SNMetamorph** [comment]
> Actually, we probably can just write clc_stringcmd directly to netchan instead. =/

Without even using cmdbuffer? Why not

**Comment by a1batross** [comment]
Fixed in dca637d4
